core/server/common/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -38,6 +45,10 @@
  * Utility for checking Alluxio environment.
  */
 public final class ValidateEnv {
+  public static final Option HADOOP_CONF_DIR_OPTION =
```

Comment:
Can we query the additional options from each validation task? e.g. `registerTask` will get a list of options to be added to the option builder.

Reply:
Done.